taxonomy/management/commands/refresh_course_skills.py

```diff
@@ -61,11 +70,39 @@ def handle(self, *args, **options):
         """
         Entry point for management command execution.
         """
-        if options['course'] == [] and options['args_from_database'] is False:
-            parser = self.create_parser('manage.py', 'refresh_course_skills')
-            parser.error('Either course or args_from_database argument must be present')
+        if not (options['args_from_database'] or options['all'] or options['course']):
```

Comment:
Is it possible that we can do this kind of validation in `parser.add_argument` instead of here? If we add another argument in the future then I think we also need to update this if condition also.

Reply:
I think there is not any specific method provided by the library.
see this thread.
https://stackoverflow.com/questions/6722936/python-argparse-make-at-least-one-argument-required
@sameenfatima78 @muhammad-ammar

Reply:
@sameenfatima78 Not actually. As there are arguments that are not required. For example --commit is optional.